src/main/resources/MxrdrBundle.properties

```diff
@@ -0,0 +1,15 @@
+workflow.success.xds=Automatic analysis with XDS software of {0} has been successfully accomplished.
+workflow.fail.xds=Automatic analysis with XDS software of {0} has failed.
+mail.subject.workflow.success.xds=Automatic analysis of your dataset has been successfully accomplished
+mail.subject.workflow.fail.xds=Automatic analysis of your dataset has failed
+mail.content.workflow.success.xds=Dear User,\n Automatic analysis with XDS software of the dataset entitled “{0}” has been successfully accomplished.\n You can view the detailed results of the analysis on the dataset’s page ({1}).\n You may contact us for support at {2}.
+mail.content.workflow.fail.xds=Dear User,\n Automatic analysis with XDS software of the dataset entitled “{0}” has failed.\n You can view the detailed results of the analysis on the dataset’s page ({1}).\n You may contact us for support at {2}.
+
+dataset.analysisTab.embargo.message=Results of the automatic analysis with XDS of this version of the dataset will be available after the end of the embargo period, i.e. from {0}.</p>
+dataset.analysisTab.artifacts.message=You can find more details in the files available for download:
+dataset.analysisTab.analysis.in.progress.message=Automatic analysis with XDS of this version of the dataset is in progress.
+dataset.analysisTab.analysis.not.performed.message=Automatic analysis with XDS of this version of the dataset was not performed.
+dataset.analysisTab.analysis.failed.message=Automatic analysis with XDS of this version of the dataset failed.
+dataset.analysisTab.dataset.in.draft.message=The attempt of automatic analysis with XDS will start after this version of this dataset is published.
+
+
```

Comment:
What's the reason behind having extra properties file?

Reply:
We need default locale - when user starts with his default locale (for example - PL) then error 500 occurs